spec/requests/api/v0/manifests_search_spec.rb

```diff
@@ -1,35 +1,31 @@
 require 'rails_helper'
-require_relative '../../db/chores/populator'
 require "queryparams"
 
 describe "search", elasticsearch: true do
+  before(:each) do
+    Manifest.rebuild_index
```

Comment:
it's a bit like database cleaner: resetting the storage to a known good state. Would you like to see that inlined as well?